teleport.js

```diff
@@ -134,6 +134,23 @@ function getPlayers() {
   return players
 }
 
+/*
+ * get maps containing an item with a certain name.
+ */
+function getMapsWithItemName(itemName) {
+  let mapsWithItemName = [];
+  wrapper((gameSpace) => {
+    const mapKeys = Object.keys(gameSpace.maps);
+    const maps = gameSpace.maps;
+    mapKeys.forEach(key => {
+      if (maps[key].objects.filter(o => (o._name || '').includes(itemName)).length > 0) {
+        mapsWithItemName.push(gameSpace.maps[key]);
+      }
+    })
```

Comment:
`mapsWithItemName =  mapKeys.filter(key => map[key].objects.filter(o => (o._name || '').include(itemName)).length > 0)` should be sufficient

Reply:
and you can also empty initialize mapsWIthItemName